frontend/src/components/Layout/TrainingDS/DatasetEditor/AOI.js

```diff
@@ -114,9 +114,9 @@ const AOI = (props) => {
   return (
     <>
       <Grid item md={12} className="card" marginBottom={1}>
-        <Tooltip title="For each AOI, we need to make sure labels inside it are alighed and complete to acheive best model accuracy">
+        <Tooltip title="For each AOI, we need to ensure that all buildings are mapped and correctly aligned.">
```

Comment:
Currently its building but label might not represent only buildings later stage , I wonder how we can communicate this

Reply:
This is surprisingly difficult. We cannot say "all map features" because we cannot handle all kinds of map features. Saying "all map features that fAIr can handle" is too complex and subtle. Can we stay with "buildings" and hope it is updated when needed? I think that even if roads and water flows are added, "buildings" is more useful than "labels".